CONTRIBUTING.md

```diff
@@ -14,6 +27,10 @@
 * fork and clone the repository to your machine
 * `bundle` to install ruby gems
 
+Optional: If you used Homebrew to install Git Town, you will need to ensure the development repository
+is placed before the Homebrew directory in your PATH variable. E.g: `... PATH=~/your/dev/path/git-town/src: ...`
```

Comment:
yep. I had too many times where I was working on git town and then needed to sync or something and had it break.